app/view/panel/document/DocumentTreeController.js

```diff
@@ -23,25 +24,26 @@ Ext.define('MoMo.client.view.panel.rbma.RbmaTreeController', {
     },
 
     /**
-     * Set store for the RBMA document tree
+     * Set store for the document tree
      */
-    setRbmaStoreAndRootNode: function() {
+    setDocumentStoreAndRootNode: function() {
         var me = this,
```

Comment:
MINOR: Would be cleaner if you won't mix comma seperated variable declaration with the semicolon way you did before.